deafrica/data/chirps.py

```diff
@@ -45,6 +51,9 @@ def download_and_cog_chirps(
         # Set up a daily process
         in_file = f"chirps-v2.0.{year}.{month}.{day}.tif.gz"
         in_href = DAILY_URL_TEMPLATE.format(year=year, in_file=in_file)
+        if not check_for_compression(in_href):
+            in_file = f"chirps-v2.0.{year}.{month}.{day}.tif"
+            in_href = DAILY_URL_TEMPLATE.format(year=year, in_file=in_file)
         in_data = f"/vsigzip//vsicurl/{in_href}"
```

Comment:
If it's not compressed, we need to remove the `/vsigzip/` too.

Reply:
I have updated this.